pexec/managed_process.go

```diff
@@ -39,7 +39,7 @@ type ManagedProcess interface {
 
 // NewManagedProcess returns a new, unstarted, from the given configuration.
 func NewManagedProcess(config ProcessConfig, logger utils.ZapCompatibleLogger) ManagedProcess {
-	logger = utils.Sublogger(logger, fmt.Sprintf("process.%s_%s", config.ID, config.Name))
+	logger = utils.Sublogger(logger, config.ID)
```

Comment:
`config.ID` is the name of the module and `config.Name` is the path to the module executable per how we create the `ProcessConfig` in the module manager code. The change I have here changes logs like:
```
2024-11-19T15:17:06.065Z	ERROR	rdk.modmanager.process.SimpleModule_/home/benjirewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdErr	pexec/managed_process.go:275	
\_ A message from the module to STDERR
2024-11-19T15:17:06.065Z	INFO	rdk.modmanager.process.SimpleModule_/home/benjirewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:277	
\_ A message from the module to STDOUT
```
to
```
2024-11-19T15:10:04.491Z	ERROR	rdk.modmanager.SimpleModule.StdErr	pexec/managed_process.go:275	
\_ A message from the module to STDERR
2024-11-19T15:10:04.491Z	INFO	rdk.modmanager.SimpleModule.StdOut	pexec/managed_process.go:277	
\_ A message from the module to STDOUT
```

It's not perfect, but I find it much easier to read, and I do not think the `process` prefix or the executable path are useful.

Reply:
Agreed, removing the executable path is not helpful. Just leave a comment that the `config.ID` maps to the module name in the robot config.